src/utils.ts

```diff
@@ -116,3 +116,13 @@ type Truthy<T> = T extends false | '' | 0 | null | undefined ? never : T // from
 export function truthy<T>(value: T): value is Truthy<T> {
   return Boolean(value)
 }
+
+export function jsoncParse(data: string) {
+  try {
+    return new Function('return ' + strip(data).trim())()
+  } catch (_) {
```

Comment:
```suggestion
  } catch {
```
remove redundant parentheses